roles/kubernetes-apps/policy_controller/calico/templates/calico-kube-controllers.yml.j2

```diff
@@ -40,11 +40,22 @@ spec:
             requests:
               cpu: {{ calico_policy_controller_cpu_requests }}
               memory: {{ calico_policy_controller_memory_requests }}
+{% if calico_version is version('v3.19.0', '>=') %}
+          livenessProbe:
+            exec:
+              command:
+              - /usr/bin/check-status
+              - -l
+            periodSeconds: 10
+            initialDelaySeconds: 10
+            failureThreshold: 6
+{% endif %}
```

Comment:
Is the above change for Calico v3.19.0?  not for eBPF?
I could not find the relationship between the above change and eBPF.

Reply:
This is for v3.19.0 not strictly for eBPF. This liveness probe mechanism is not available in 3.18 and prior versions.

Reply:
@cristicalin Thanks for your clear explanation, I got it.
Then it is better to merge https://github.com/kubernetes-sigs/kubespray/pull/7630 before this.